assets/application.json

```
@@ -209,6 +209,12 @@
               "maximum": 100,
               "title": "Taxa sobre o subtotal",
               "description": "Adicional/desconto percentual sobre o valor subtotal da compra"
+            },
+            "delivery_instructions": {
+              "type": "string",
+              "maxLength": 1000,
+              "title": "Instruçoes de entrega",
+              "description": "Insira informações detalhadas para seu cliente retirar o pedido"
```

```suggestion
              "description": "Insira informações adicionais para retirada ou entrega do pedido"
```

Não necessariamente é retirada aqui...